src/components/EpicView/helpers/epicViewHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import { Issue } from "../../../../types";
+
+export function filtersearch(query :string, epics : Issue[]) {
+  const searchstring = query.toLowerCase().trim();
+  return epics.filter((item) => item.summary.toLowerCase().includes(searchstring)
+    || searchstring === ""
+    || item.issueKey.toLowerCase().includes(searchstring)
+    || (item.assignee?.displayName?.toLowerCase().includes(searchstring))
+    || (item.creator?.toLowerCase().includes(searchstring))
+    || !(item.labels?.map((label) => label.toLowerCase()).filter((label) => label.includes(query)).length === 0)
+    || (item.status?.toLowerCase().includes(searchstring)));
+}
```

Comment:
![image](https://github.com/MaibornWolff/ProjectCanvas/assets/78490564/a44af710-c83e-43a3-ad96-d97aebab1795)

Several things:
- Please correct the `:` spacing in the method descriptor
- Please remove excess `(` and `)` around conditions (e.g. line 8)
- Please camelcase the name of this function correctly

Reply:
oh, didn´t notice them. Also got the label filter search from Backlog, using some is way more efficient than doing filter + map. Thanks!